src/PhpBrew/Command/AppCommand.php

```diff
@@ -18,11 +18,6 @@ public function brief()
         return 'php app store';
     }
 
-    public function options($opts)
-    {
-        $opts->add('l|list', 'Show app list.');
-    }
-
```

Comment:
I realize that this is a breaking change but I'd rather document it in the release notes than implement the handler of this option that would emit a deprecation message.